packages/libplist.rb

```diff
@@ -15,11 +15,9 @@ class Libplist < Package
   depends_on 'glib'
 
   def self.build
-    system './autogen.sh CC=gcc CXX=g++'
-    system './configure',
-      "--prefix=#{CREW_PREFIX}",
-      "--libdir=#{CREW_LIB_PREFIX}",
-      '--disable-dependency-tracking'
+    system "./configure CC=gcc CXX=g++ \
```

Comment:
We still need to run `autogen.sh` since there is no `configure` available from the source.